lib/bindConnectors/formatInputBody/index.js

```diff
@@ -45,6 +45,11 @@ function formatInputValue(value, schemaProperty) {
 
   schemaProperty = schemaProperty || {};
 
+  // Ignore value for oneOfs
```

Comment:
You also need to add something like this for `items` when type is `array`, because `items` can either be
- A single schema object
```javascript
items: {
   type: 'string'
}
```
- An array of schema objects
```javascript
items: [
    {
       type: 'string'
    },
    {
       type: 'number'
    }
]
```
- **A `oneOf` array of schema objects**
```javascript
items: {
    oneOf:  [
        {
            type: 'string'
        },
        {
            type: 'number'
        }
    ]
}
```